data/plugins/googleBigQuery.ts

```diff
@@ -1,7 +1,7 @@
 import { Plugin } from "./index";
 
 export const GoogleBigQuery: Plugin = {
-  name: "Google BigQuery",
```

Comment:
@andyjih this would change the name of the plugin on the other pages as well.  What do you think?

Reply:
I thought the same thing, but I looked around and all looked fine to me.